docs/platforms/javascript/guides/angular/index.mdx

```diff
@@ -1,80 +1,83 @@
+at least Angular 17.
 
-Sentry captures data by using an SDK within your application’s runtime.
+</Expandable>
 
-```bash {tabTitle:npm}
-npm install @sentry/angular --save
-```
+The wizard then guides you through the setup process, asking you to enable additional (optional) Sentry features for your application beyond error monitoring.
 
-```bash {tabTitle:yarn}
-yarn add @sentry/angular
-```
+<PlatformContent includePath="getting-started-features-expandable" />
 
-```bash {tabTitle:pnpm}
-pnpm add @sentry/angular
-```
+This guide assumes that you enable all features and allow the wizard to create an example page and route. You can add or remove features at any time, but setting them up now will save you the effort of configuring them manually later.
 
-### Angular Version Compatibility
+<Expandable title="What does the installation wizard change inside your application?">
 
-In its current major version, the Sentry Angular SDK only supports Angular 14 and newer.
+- Adds a `Sentry.init()` call into your `main.ts` file.
+- Registers the Sentry `ErrorHandler` and `TraceService` and in your `app.config.ts` file.
+- Creates `.sentryclirc` with an auth token to upload source maps (this file is automatically added to `.gitignore`)
+- Walks you through enabling source maps upload when making a production build, locally as well as in CI.
+- Adds an example component to your application to help verify your Sentry setup
 
-If you're using an older version of Angular, you also need to use an older version of the SDK. See the table below for compatibility guidance:
+</Expandable>
 
-| Angular version | Recommended Sentry SDK                                                                                  |
-| --------------- | ------------------------------------------------------------------------------------------------------- |
-| 14 and newer    | `@sentry/angular`                                                                                       |
-| 12 or 13        | `@sentry/angular-ivy@^7` (see [Note](#what-is-sentryangular-ivy)) \*                                    |
-| 10 or 11        | `@sentry/angular@^7` \*                                                                                 |
-| 9 and below     | `@sentry/angular@^6` \*                                                                                 |
-| AngularJS/1.x   | `@sentry/browser@^6` with the [AngularJS integration](/platforms/javascript/guides/angular/angular1) \* |
+## Step 2: Verify
 
-\* These versions of the SDK are no longer maintained or tested. Version 7 might still receive bug fixes but we don't guarantee support.
+If you haven't tested your Sentry configuration yet, let's do it now. You can confirm that Sentry is working properly and sending data to your Sentry project by using the example component created by the installation wizard:
```

Comment:
Shouldn't this be "send" here? Because of "You can confirm" --> "You can ... send"
```suggestion
If you haven't tested your Sentry configuration yet, let's do it now. You can confirm that Sentry is working properly and send data to your Sentry project by using the example component created by the installation wizard:
```

Reply:
Ah I think this sentence isn't clear on the subject. technically it is correct because you can verify that *sentry* is working and sending data but let's re-word and simplify this a bit

```suggestion
If you haven't tested your Sentry configuration yet, let's do it now. You can confirm that the Sentry SDK is sending data to your Sentry project by using the example component created by the installation wizard:
```

Reply:
Ahh yeah that makes more sense :D

I see this wording is also used on the Next.js and Nuxt pages - we should also update it here. Do you want to do this in this PR?